.github/SUPPORT.md

```diff
@@ -0,0 +1,32 @@
+Support Guide
+===
+
+Welcome to the support guide! What type of help are you looking for?
+
+## I need help understanding how the app works
+If you want to learn how use a specific feature of the application, please see our [user guide](https://help.openboxes.com/), which describes the primary features of the app in non-technical terms.
+
+## I need help running/deploying the app
+Have you read through our [installation documentation](http://docs.openboxes.com/en/latest/installation/)? The documentation outlines all the steps that a system administrator should need to follow to install, configure, update, and deploy the application to a production server.
+
+If you're running into an issue not described in our documentation, take a look through the [support forums](https://openboxes.uservoice.com/forums/163294-general) to see if someone has encountered a similar problem. If you're still stuck, we encourage you to create a new forum post. Please make sure to let us know the step of the installation process that you're stuck on.
+
+## I think I found a bug in the app
+If you've found a bug in OpenBoxes, take a look through the [existing reported issues](https://github.com/openboxes/openboxes/issues) to see if the bug has already been reported, and if not, please [open a new issue](https://github.com/openboxes/openboxes/issues/new).
+
+If you've uncovered a security vulnerability in the app, we ask that instead of creating an issue, you please email us at support@openboxes.com with details about the vulnerability so that we can protect our user base.
+
+When creating a new GitHub issue, be sure to add enough context in the issue description for the team to be able to properly investigate. At a minimum, please include the following:
+
+1. The version of OpenBoxes you're using. You can find the version at the bottom of the webpage.
+2. The operating system and browser you're using.
+3. The steps you performed to produce the bug.
+4. What you expected to see.
+5. What you saw instead.
+6. Screenshots and videos are extremely helpful! Please include some if possible.
+
+## I want to speak with the team directly
+
+Our team actively monitors the [support forums](https://openboxes.uservoice.com/forums/163294-general) for new posts, but if you want to reach us directly for any reason, you can email us at support@openboxes.com.
```

Comment:
Nice! Way to include the user voice forum. I would advocate for linking to Slack, the Community (community.openboxes.com) and/or Github Discussions (https://github.com/openboxes/openboxes/discussions) for these types of discussions. But I'm open to using UserVoice if that would be preferred.